src/org/labkey/test/pages/user/ClonePermissionsPage.java

```diff
@@ -0,0 +1,64 @@
+package org.labkey.test.pages.user;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Map;
+
+public class ClonePermissionsPage extends LabKeyPage<ClonePermissionsPage.ElementCache>
+{
+    public ClonePermissionsPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ClonePermissionsPage beginAt(WebDriverWrapper webDriverWrapper, Integer userId)
+    {
+        Map<String, String> params = Map.of("userId", userId.toString(), "schemaName", "core", "query.queryName", "Users", "targetUser", userId.toString());
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("security", "clonePermissions", params));
+        return new ClonePermissionsPage(webDriverWrapper.getDriver());
+    }
+
+    public String getWarningMessage()
+    {
+        return elementCache().warningMsg.getText();
+    }
+
+    public ClonePermissionsPage setCloneUser(String value)
+    {
+        elementCache().cloneUser.set(value);
+        return this;
+    }
+
+    public ShowUsersPage clonePermission()
+    {
+        clickAndWait(elementCache().clonePermissionBtn);
+        return new ShowUsersPage(getDriver());
+    }
```

Comment:
This should verify that there weren't any errors. Otherwise, tests will call this then fail on subsequent checks without much information. There should also be a matching `clonePermissionExpectingError()` that verifies that there **is** an error.